driver-moab/main/cplcomp_exchange_mod.F90

```diff
@@ -1204,14 +1199,12 @@ subroutine cplcomp_moab_Init(infodata,comp)
                write(logunit,*) subname,' error in defining tags seq_flds_dom_fields on atm on coupler '
                call shr_sys_abort(subname//' ERROR in defining tags ')
             endif
-            if ( trim(atm_mesh) /= 'none' ) then
-               ! also, frac, area, aream, masks has to come from atm mphaid, not from domain file reader
-               ! this is hard to digest :(
-               tagname = 'area:aream:frac:mask'//C_NULL_CHAR
```

Comment:
maybe remove aream, it will be sent back from coupler, later

Reply:
done